the-metalworks/cosmere-rpg
# chore(welcome message): update text to point towards the new repo and discord server

**Type**  
What type of pull request is this? (e.g., Bug fix, Feature, Refactor, etc.)


**Description**  
This PR updates the welcome message text to include the new repo and point towards the new discord server. Additionally it changes the message sender to "The Metalworks".

**Related Issue**  
Closes #162 

**How Has This Been Tested?**  
1. Update the `INTERNAL_FIRST_CREATION` setting to `true`
2. Check the welcome message

**Screenshots (if applicable)**  
![image](https://github.com/user-attachments/assets/c312afe9-5349-4f58-bb76-f493d8568435)

**Checklist:**
